Remove weak package

blowery opened the pull request:
It was only used for leak detection with jest and isn't needed day to day,
Remove it for now to cut down on build times a touch.

jsnajdr reviewed (approved):
I say :+1: although we should normally protect the weak.

sirreal reviewed (approved):
🚢

blowery committed:
Remove weak package

It was only used for leak detection with jest and isn't needed day to day.
Remove it to cut down on build time a touch.
